pkgs/tools/nix/nixos-option/src/nixos-option.cc

```diff
@@ -40,8 +54,8 @@ struct Context
         : state(state), autoArgs(autoArgs), optionsRoot(optionsRoot), configRoot(configRoot),
           underscoreType(state.symbols.create("_type"))
     {}
-    EvalState & state;
-    Bindings & autoArgs;
+    EvalState & state; // NOLINT(cppcoreguidelines-avoid-const-or-ref-data-members)
+    Bindings & autoArgs; // NOLINT(cppcoreguidelines-avoid-const-or-ref-data-members)
```

Comment:
I think we can disable this globally. `cppcoreguidelines` generally have too many noisy checks. If we want this class to be assignable then `std::reference_wrapper` will do fine.